docs/dump_javadump.md

```diff
@@ -925,7 +927,242 @@ Help > Install New Software > Work with "IBM Diagnostic Tool Framework for Java"
+
+When a `NativeOutOfMemoryError` occurs, a Java dump is generated by default. The first section of the file (TITLE) tells you that a systhrow event triggered the Java dump as a result of an OOM (`java/lang/OutOfMemoryError`) for native memory.
+
+```
+0SECTION       TITLE subcomponent dump routine
+NULL           ===============================
+1TICHARSET     UTF-8
+1TISIGINFO     Dump Event "systhrow" (00040000) Detail "java/lang/OutOfMemoryError" "native memory exhausted" received
+1TIDATETIME    Date: 2018/09/14 at 15:49:55:887
+1TINANOTIME    System nanotime: 3636862054495675
+1TIFILENAME    Javacore filename:    /home/cheesemp/test/javacore.20180914.154814.19708.0003.txt
+1TIREQFLAGS    Request Flags: 0x81 (exclusive+preempt)
+1TIPREPSTATE   Prep State: 0x104 (exclusive_vm_access+trace_disabled)
+```
+
+Sometimes, the current thread is responsible for causing the `NativeOutOfMemoryError`. Information about the current thread can be found in the THREADS section, as shown in the following output.
+
+```
+0SECTION       THREADS subcomponent dump routine
+NULL           =================================
+NULL
+1XMPOOLINFO    JVM Thread pool info:
+2XMPOOLTOTAL       Current total number of pooled threads: 16
+2XMPOOLLIVE        Current total number of live threads: 16
+2XMPOOLDAEMON      Current total number of live daemon threads: 15
+NULL            
+1XMCURTHDINFO  Current thread
+3XMTHREADINFO      "main" J9VMThread:0xB6C60C00, omrthread_t:0xB6C049D8, java/lang/Thread:0xB55E3C10, state:R, prio=5
+3XMJAVALTHREAD            (java/lang/Thread getId:0x1, isDaemon:false)
+3XMTHREADINFO1            (native thread ID:0x4CFD, native priority:0x5, native policy:UNKNOWN, vmstate:R, vm thread flags:0x00001020)
+3XMTHREADINFO2            (native stack address range from:0xB6D4E000, to:0xB754F000, size:0x801000)
+3XMCPUTIME               CPU usage total: 3.654896026 secs, current category="Application"
+3XMHEAPALLOC             Heap bytes allocated since last GC cycle=0 (0x0)
+3XMTHREADINFO3           Java callstack:
+4XESTACKTRACE                at sun/misc/Unsafe.allocateDBBMemory(Native Method)
+4XESTACKTRACE                at java/nio/DirectByteBuffer.<init>(DirectByteBuffer.java:127(Compiled Code))
+4XESTACKTRACE                at java/nio/ByteBuffer.allocateDirect(ByteBuffer.java:311)
+4XESTACKTRACE                at NativeHeapBreaker.main(NativeHeapBreaker.java:9)
+3XMTHREADINFO3           Native callstack:
+4XENATIVESTACK               (0xB6A9F5B3 [libj9prt29.so+0x3b5b3])
+4XENATIVESTACK               (0xB6A82F3E [libj9prt29.so+0x1ef3e])
+4XENATIVESTACK               (0xB6A9F11E [libj9prt29.so+0x3b11e])
+4XENATIVESTACK               (0xB6A9F216 [libj9prt29.so+0x3b216])
+4XENATIVESTACK               (0xB6A9ECE3 [libj9prt29.so+0x3ace3])
+4XENATIVESTACK               (0xB6A82F3E [libj9prt29.so+0x1ef3e])
+4XENATIVESTACK               (0xB6A9EC57 [libj9prt29.so+0x3ac57])
+4XENATIVESTACK               (0xB6A9A9CD [libj9prt29.so+0x369cd])
```

Comment:
Can we `<snip>` some of the frames in the middle here?  It would make it more obvious which frames matter

Reply:
Good idea. Amended.